packages/wonder-blocks-button/components/button.md

```diff
@@ -274,7 +274,7 @@ const styles = StyleSheet.create({
 </View>
 ```
 
-Buttons can do client-side navigation if `clientNav` is `true`.
+Buttons do client-side navigation by default, if React Router exists:
```

Comment:
We should include an example of how to avoid client-side navigation within a router if necessary.  Also, I wonder what happens if people set `target="_blank"` on a button with an `href`. 🤔

Reply:
I added an example of a link with `skipClientNav` - but yeah, maybe we should make it so that if you set a `target` it automatically sets `skipClientNav={true}`?

Reply:
It looks like `react-router`'s `Link` supports this out of the box so `skipClientNav` isn't necessary when `taget="_blank"` is set.  https://github.com/ReactTraining/react-router/pull/2193